### ISSUE TITLE
Gradle Build shows additional logs

### ISSUE THREAD

**Issue by hasithaa** [ballerina-platform/ballerina-lang#18276]
**Description:**
Following logs are visible now. This is observed after alpha3 pack. Env is windows.
```
 Task :ballerina-lang:decimal:createBalo
Compiling source
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/C:/Users/hasitha/.m2/repository/org/slf4j/slf4j-log4j12/1.7.25/slf4j-log4j12-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/C:/Users/hasitha/.gradle/caches/modules-2/files-2.1/org.ops4j.pax.logging/pax-logging-api/1.10.0/56cfd0505e45d13743fb594935a19915688b8e32/pax-logging-api-1.10.0.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
[io.netty.util.internal.logging.InternalLoggerFactory] : Using Log4J as the default logging framework
[io.netty.util.ResourceLeakDetector] : -Dio.netty.leakDetection.level: simple
[io.netty.util.ResourceLeakDetector] : -Dio.netty.leakDetection.targetRecords: 4
[io.netty.util.internal.PlatformDependent] : Platform: Windows
[io.netty.util.internal.PlatformDependent0] : -Dio.netty.noUnsafe: false
[io.netty.util.internal.PlatformDependent0] : Java version: 8
[io.netty.util.internal.PlatformDependent0] : sun.misc.Unsafe.theUnsafe: available
[io.netty.util.internal.PlatformDependent0] : sun.misc.Unsafe.copyMemory: available
[io.netty.util.internal.PlatformDependent0] : java.nio.Buffer.address: available
[io.netty.util.internal.PlatformDependent0] : direct buffer constructor: available
[io.netty.util.internal.PlatformDependent0] : java.nio.Bits.unaligned: available, true
[io.netty.util.internal.PlatformDependent0] : jdk.internal.misc.Unsafe.allocateUninitializedArray(int): unavailable prior to Java9
[io.netty.util.internal.PlatformDependent0] : java.nio.DirectByteBuffer.<init>(long, int): available
[io.netty.util.internal.PlatformDependent] : sun.misc.Unsafe: available
[io.netty.util.internal.PlatformDependent] : -Dio.netty.tmpdir: C:\Users\hasitha\AppData\Local\Temp (java.io.tmpdir)
[io.netty.util.internal.PlatformDependent] : -Dio.netty.bitMode: 64 (sun.arch.data.model)
[io.netty.util.internal.PlatformDependent] : -Dio.netty.maxDirectMemory: 3790077952 bytes
[io.netty.util.internal.PlatformDependent] : -Dio.netty.uninitializedArrayAllocationThreshold: -1
[io.netty.util.internal.CleanerJava6] : java.nio.ByteBuffer.cleaner(): available
[io.netty.util.internal.PlatformDependent] : -Dio.netty.noPreferDirect: false
[io.netty.buffer.AbstractByteBuf] : -Dio.netty.buffer.checkAccessible: true
[io.netty.buffer.AbstractByteBuf] : -Dio.netty.buffer.checkBounds: true
[io.netty.util.ResourceLeakDetectorFactory] : Loaded default ResourceLeakDetector: io.netty.util.ResourceLeakDetector@5e3d57c7
        ballerina/lang.decimal
```

**Steps to reproduce:**

**Affected Versions:**

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by Shan1024** [comment]
These logs are visible in MacOS environment as well. I think I saw these prior to alpha3 pack as well.

**Comment by pubudu91** [comment]
This was fixed with PR #19543. Hence closing this.
